code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -136,7 +136,7 @@
 	overdose_threshold = 100 //no chugging
 
 /datum/reagent/medicine/cryoxadone/on_mob_life(mob/living/carbon/M)
-	var/power = -0.00006 * (M.bodytemperature ** 2) + 6
+	var/power = -0.00006 * (M.bodytemperature ** 2) + 3
```

Comment:
This part of the equation needs to be halved too or it will cause damage between 223-273 kelvin instead of healing it
```suggestion
	var/power = -0.00003 * (M.bodytemperature ** 2) + 3
```